### PR TITLE
Added THEME toogle bar

### PR DESCRIPTION
**Pull request by Sid911**
#### Changes proposed in this pull request:

- Added theme toggle bar to the Firefox extension 

### Before
image
### After
image
image

### CODE REVIEW

**Comment by neeraj3029** [comment]
@Sid911 , please mention the issue number in your PR message. And please look at PR #119 , I guess it is same as yours.

**Comment by Sid911** [comment]
@neeraj3029  It is actually a gci task.... so I don't know if its an issue or not

**Comment by Sid911** [comment]
@akshatnitd I have a big problem.... how do I load my extension , I can't find a way because it only taks xpi files

**Comment by Sid911** [comment]
Working on it sir

**Comment by Sid911** [comment]
@akshatnitd Done

**Comment by Sid911** [comment]
@pranav1698 Done.... I fixed them

**Comment by Sid911** [comment]
@pranav1698 Done

File: src/scripts/main.js [hunk 1]
```diff
@@ -694,3 +694,20 @@ function clearMessageHistory() {
...
+		browser.storage.sync.set({ theme: "dark" });
+		theme = "dark";
+		applyTheme();
+	} 
+	else {
+		browser.storage.sync.set({ theme: "light" });
+		theme = "light";
+		applyTheme();
+	}
+}
```
Comment (Ryan10145) [inline src/scripts/main.js]: Can you add a new line at the end of the file?

**Comment by Sid911** [comment]
@Ryan10145. I am not sure why we should do that.... But I will do it nonetheless

**Comment by tabesin** [comment]
Approved.

Pls check line 90 in chatbot.html.

**Comment by Sid911** [comment]
@tabesin  Yes sir... I will delete that ... I think it was there from before

**Comment by jogendra** [comment]
@tabesin please merge

**Pull request merged**